merlin/models/tf/models/ranking.py

```diff
@@ -235,3 +236,131 @@ def DeepFMModel(
+        wide model, and the model would become a pure deep model, if specified, only features
+        in wide_schema would be sent to wide model
+    deep_schema : Optional[Schema]
+        The 'Schema' of input features for deep model, by default all features would be sent to
+        deep model. deep_schema and wide_schema could contain the same features
+    wide_preprocess : Optional[Block]
+        Transformation block for preprocess data in wide model. Such as CategoricalOneHot,
+        CategoryEncoding, HashedCross, and HashedCrossAll, please note the schema of transformation
+        block should be the same as the wide_schema.  By default None.
+        For example:
+            ```python
+            # CategoricalOneHot as preprocess for wide model
+            import merlin.models.tf as ml
+            model = ml.benchmark.WideAndDeepModel(
+                schema = schema,
+                wide_schema=wide_schema,
+                deep_schema=deep_schema,
+                wide_preprocess = ml.CategoricalOneHot(wide_schema)
+                deep_block=ml.MLPBlock([32, 16]),
+                prediction_tasks=ml.BinaryClassificationTask("click"),
+            )
+
+            # HashedCross as preprocess for wide model
+            model = ml.benchmark.WideAndDeepModel(
+                schema = schema,
+                wide_schema=wide_schema,
+                deep_schema=deep_schema,
+                wide_preprocess = ml.HashedCross(wide_schema, num_bins=1000),
+                deep_block=ml.MLPBlock([32, 16]),
+                prediction_tasks=ml.BinaryClassificationTask("click"),
+            )
+            ```
+    prediction_tasks: Optional[Union[PredictionTask, List[PredictionTask], ParallelPredictionBlock]
+        The prediction tasks to be used, by default this will be inferred from the Schema.
+
+    Returns
+    -------
+    Model
+
+    """
+
+    prediction_tasks = parse_prediction_tasks(schema, prediction_tasks)
+
+    if not wide_schema:
+        warnings.warn("If not specify wide_schema, NO feature would be sent to wide model")
+        wide_schema = None
+
+    if not deep_schema:
+        deep_schema = schema
+
+    if not deep_input_block:
+        if len(deep_schema) > 0:
+            deep_input_block = InputBlockV2(
+                deep_schema,
+                **kwargs,
+            )
+    deep_body = deep_input_block.connect(deep_block).connect(
+        MLPBlock([1], no_activation_last_layer=True)
+    )
+
+    if not wide_input_block:
+        if len(wide_schema) > 0:
+            wide_input_block = ParallelBlock(
+                TabularBlock.from_schema(schema=wide_schema, pre=wide_preprocess),
+                is_input=True,
+                aggregation="concat",
+            )
+    wide_body = wide_input_block.connect(MLPBlock([1], no_activation_last_layer=True))
+
+    branches = {"wide": wide_body, "deep": deep_body}
+    wide_and_deep_body = ParallelBlock(branches, aggregation="element-wise-sum")
```

Comment:
In the W&D paper they say "The wide component and deep component are combined using a weighted sum of their output log odds as the prediction". But I have checked the [TF implementation](tf.estimator.DNNLinearCombinedClassifier ) and they just sum the logits of both heads, so I think we are fine here